documentation/API_documentation/QoD_Stable_Bandwidth_API.md

```diff
@@ -0,0 +1,180 @@
+# Overview
+
+<span class="colour" style="color:rgb(0, 0, 0)">The Quality-On-Demand (QoD) API for stable throughput provides programmable interface for developers and other users (capabilities consumers) to specify stable data-transfer throughput managed by Telco networks without necessity to have in-depth knowledge of the 4G/5G system complexity by abstracting the internal complexity of telecom systems [1].</span>
+
+## 1\. Introduction
+
+There are many real-world use cases which could exploit the potential of programmable networks. Several of these use case implementations are still work in progress or have not been operationalized as developers have no means to configure or request stable latency or bandwidth during congestion from a mobile network operator which is critical. Use cases from industrial (IoT), VR/Gaming, broadcasting, autonomous driving and many others fall in such a category since they demand network communication quality and are sensitive to any change in transmission conditions. 
+
+The QoD throughput API offers the application developers and users the capability to request for stable throughput for a specified App-Flow(s) between application clients and backend services (Application Servers). The developer has a pre-defined set of QoS_Profiles which he could choose from depending on his throughput requirements. 
+
+<img src="./resources/QoD_bandwidth_overview.PNG" alt="QoD_BM" title="QoD Bandwidth API Overview">
+
+
+## 2\. Quick Start
+
+The usage of the stable throughput API is based on QoS sessions, which can be created (based on available QoS profiles), queried and deleted.
+The deletion of a requested session can be triggered by the user or can be triggered automatically. The automatic process is triggered either when the user specified duration has reached its limit or default session expiration time has been reached (currently set to 24hrs). 
```

Comment:
"its limit or default" -> "its limit, or default"

Reply:
Done in [1fd0cfe](https://github.com/camaraproject/QualityOnDemand/pull/17/commits/1fd0cfe1c718e6362cfb0f1536790f28fb003d8e)